ft_memset.c

```diff
@@ -0,0 +1,28 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_memset.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ekorley <marvin@42.fr>                     +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2016/09/22 07:46:25 by ekorley           #+#    #+#             */
+/*   Updated: 2016/09/22 13:25:08 by ekorley          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+
+void	*ft_memset(void *b, int c, size_t len)
+{
+	size_t			i;
+	unsigned char	*ptr;
+
+	ptr = b;
+	i = 0;
+	while (i < len)
+	{
+		ptr[i] = c;
```

Comment:
Do you want to put an actual `int` type into the destination memory?  I think you want to cast `c` to be an `unsigned char`.